project/src/main/java/caselab/service/users/ApplicationUserService.java

```diff
@@ -60,4 +58,10 @@ private void updatePassword(ApplicationUser userToUpdate, String password) {
             userToUpdate.setHashedPassword(hashedPassword);
         }
     }
+
+    private EntityNotFoundException userNotFound(Long id) {
+        return new EntityNotFoundException(
+            messageSource.getMessage("user.not.found", new Object[] {id}, Locale.getDefault())
```

Comment:
Тут тоже дефолтное сообщение не помешает
```suggestion
            messageSource.getMessage("user.not.found", new Object[] {id}, "user.not.found", Locale.getDefault())
```